### DISCUSSION TITLE
We're moving community discussions

### DISCUSSION THREAD

**Discussion by kyouko-taiga** [hylo-lang/hylo-lang.github.io#70] [Announcements]
Hey everyone.

We'll be hosting the community discussions from another repository, so we've transferred all discussions over https://github.com/orgs/val-lang/discussions.

**Comment by dabrahams** [comment]
Perhaps the back link is no longer needed now that you've moved all the threads (nice job, BTW!)?  Also, should I be allowed to make comments here?

**Reply by kyouko-taiga** [reply]
Right!
